src/jobs/js_build_job.yml

```diff
@@ -13,46 +41,23 @@ steps:
       at: .
   - when:
       condition:
-        equal: [ true, << parameters.require_env_cleanup >> ]
+        equal: [true, << parameters.require_env_cleanup >>]
       steps:
         # to be sure that local env is not used in production
         - run:
-            name: Delete .env
             command: rm .env
+            name: Delete .env
   - when:
       condition:
-        equal: [ nextjs, << parameters.app_type >> ]
+        equal: [nextjs, << parameters.app_type >>]
       steps:
         - run:
-            name: Generate build env
             # expose the right env variables (beta or prod) to the builder
             command: |
               APP_ENV=<< parameters.build_env >> yarn run generate:build-env
+            name: Generate build env
```

Comment:
It feels strange to have name after a command. It is often a description of a command and improves readability if its before a command.